projects/igniteui-angular/migrations/update-9_1_0/changes/selectors.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "../../common/schema/selector.schema.json",
+    "changes": [
+        {
+            "type": "component",
+            "selector": "<igx-action-icon>",
+            "replaceWith": "<igx-navbar-action>"
```

Comment:
I don't think we need the angle brackets.

Reply:
done